data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2022 Aleix Pol Gonzalez <aleixpol@kde.org>
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+ Author: Aleix Pol Gonzalez <aleixpol@kde.org>
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.GlobalShortcut:
+      @short_description: GlobalShortcut portal backend interface
+
+      This this portal lets applications register global shortcuts so they can
+      act regardless of the system state upon an input event.
+
+      To use this, an app will first create a session under which its actions
+      will be collected. Then shortcuts can be bound (see Bind), listed (see ListShortcuts).
+      It's up to the portal implementation to perform the registration
+      and tracking of that shortcut throughout the session's lifespan.
+
+      Upon a shortcut trigger, Activated will be emitted until when it's not
+      triggered anymore, then Deactivated will be issued.
+  -->
+  <interface name="org.freedesktop.portal.GlobalShortcuts">
+        <!--
+        CreateSession:
+        @options: Unused so far
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session being created
+
+        Creates a global shortcuts session.
+      </variablelist>
+    -->
+    <method name="CreateSession">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="session_handle" direction="out"/>
+    </method>
+
+      <!--
+        ListShortcuts:
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @options: Unused so far
+
+        Create a global shortcuts session.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>shortcut_ids  a{sv}</term>
```

Comment:
Good catch. I forgot to update it when I changed the .impl one.

Do you think it would be more clear if it was specified `a{ss}`? I guess so since it's strings.

Reply:
`a(ss)` for a list of the tuple pair of two strings, but I don't think it's a good idea, but would rather see `a(sa{sv})` as suggested elsewhere, i.e. so that now it can be

```
[
   'water-plants': {
     'description': 'Water the plants',
   }
]
```

but later extended to 
```
[
   'water-plants': {
     'description': 'Water the plants',
     'preferred-key-binding': '<Super>+W',
   }
]
```

if such a hint would ever be added.

Reply:
> Any reason why you prefer a(sa{sv}) to a{a{ss}}?

`a{sv}` for the per binding properties so that they can remain "type safe", so that e.g. an int can be an int, a list a list, so that we don't need no encode such things into a string.

> WRT the preferred-key-binding, we'll first need to standardise these strings.

Yea, I think as long as we prepare the API to make it possible to extend, it makes sense to leave that for a follow up version.

Reply:
Or you mean `aa{sv}` instead of `a(sa{sv})`? I.e. have the `id` in the vararg as its own property. That would work too.

Reply:
> Or you mean aa{sv} instead of a(sa{sv})?

Yes, sorry. I'm not all that fluent in this types languages yet.

Reply:
I've changed all dictionaries to be a(sa{sv}) for consistency. Please review the latest changes and see if it's what you had in mind.